beets/plugins.py

```diff
@@ -60,6 +83,10 @@ def filter(self, record):
         return True
 
 
+# Typing for listeners
+CallableWithAnyKwargs = Callable[[None, Any], None]
```

Comment:
I'd prefer to give this a meaningful name:

```suggestion
Listener = Callable[[None, Any], None]
```

I feel like that would help a bit with reading the code below.

However, I also realized that what I said before about the signature of event listeners wasn't quite right. They are called with arguments by keyword only, but they don't need to be defined with keyword-only arguments. `send(...)` below also supports the case of listeners returning something (and in fact, that is being used for some events). `Callable[[None, Any], None]` also doesn't mean the correct thing (it would mean a function that takes two positional arguments, and returns `None`).

So, probably we can't do better than `Callable[..., Any]` with the current code after all. In that case, defining a type alias here doesn't seem so useful.

Reply:
You are fully correct, no idea what happened here. I guess that's why I initially omitted the extended type hints all together for listeners.  

I would keep the extra type tho for easier refactoring later on i.e.
```python
Listener = Callable[..., Any]
```